codice/ddf
# DDF-4402 Added JSON validation amd tests to Search forms

What does this PR do?
Adds JSON validation to Search forms and Result forms to prevent the backend from sending invalid JSON values to the front end.

Who is reviewing it?@Schachte
@cantstoptheunk
@gordocanchola
@adimka
@zta6

Select relevant component teams:
Ask 2 committers to review/merge the PR and tag them here.
@bdeining
@millerw8


How should this be tested?
Run full build
Create a search form that looks for anyText with a value that is surrounded with quotes (ex. "TESTING")
Create a result form with Title and description with quotes(ex. "TESTING")
Verify Search and result forms both display when you refresh their pages

#### Any background context you want to provide?
#### What are the relevant tickets?
DDF-4402(https://codice.atlassian.net/browse/DDF-4402)
#### Screenshots

#### Checklist:

#### Notes on Review Process
Please see [Notes on Review Process](https://codice.atlassian.net/wiki/spaces/DDF/pages/71946981/Pull+Request+Guidelines) for further guidance on requirements for merging and abbreviated reviews. 

#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist. 
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.
